ssn/integrated/sosa.ttl

```diff
@@ -439,12 +449,35 @@ sosa:Observation a rdfs:Class , owl:Class ;
     owl:inverseOf sosa:madeObservation ;
     rdfs:isDefinedBy sosa: .
 
+  sosa:relatedObservation rdf:type owl:ObjectProperty ;
+    rdfs:label "related observation"@en ;
+    skos:definition "Relation to an Observation."@en ;
+    schema:domainIncludes sosa:Actuation ;
+    schema:domainIncludes sosa:ActuationCollection ;
+    schema:domainIncludes sosa:ObservableProperty ;
+    schema:domainIncludes sosa:Observation ;
+    schema:domainIncludes sosa:ObservationCollection ;
+    schema:domainIncludes sosa:Platform ;
+    schema:domainIncludes sosa:Procedure ;
+    schema:domainIncludes sosa:Sample ;
+    schema:domainIncludes sosa:Sensor ;
+    schema:rangeIncludes sosa:Observation ;
+    schema:rangeIncludes sosa:ObservationCollection ;
+    rdfs:isDefinedBy sosa: .
 
 
-sosa:Actuation a rdfs:Class , owl:Class ;
+sosa:Actuation a owl:Class ;
   rdfs:label "Actuation"@en ;
-  skos:definition "An Actuation carries out an (Actuation) Procedure to change the state of the world using an Actuator."@en ;
-  rdfs:comment "An Actuation carries out an (Actuation) Procedure to change the state of the world using an Actuator."@en ;
+  skos:definition """An Actuation carries out an (Actuation) Procedure to change the state of something in the world using an Actuator. 
+  
+  In this context hasResult shall indicate the intended value of the ActuatableProperty at the completion of the Actuation
```

Comment:
"expected value" would be more suitable here. And some others pointed out, the expected value may not be reached right at the completion of the Actuation (might even never be reached).